starcheck/data/characteristics.yaml

```diff
@@ -60,8 +60,8 @@ no_starcat_oflsid:
    - RDE
    - DC_T
 
-ccd_temp_yellow_limit: -11.5
-ccd_temp_red_limit: -10.2
+ccd_temp_yellow_limit: -10.2
```

Comment:
I think the yellow limit is just used to plot another line on the ccd temperature plot at the top... which isn't used for anything.  Should probably cut that from the plotting code, but for now it needs to be set to something.